packages/ui/src/Link.tsx

```diff
@@ -6,13 +6,20 @@ import { PartialRequired } from '@hazelcast/helpers'
 import styles from './Link.module.scss'
 import { Icon } from './Icon'
 
+export type LinkKind = 'primary' | 'secondary'
+
 const sizes = {
   normal: styles.normal,
   small: styles.small,
 }
 
 type AnchorAttributes = AnchorHTMLAttributes<HTMLAnchorElement>
 
+// Common props for all link "kinds"
+type LinkCommonProps = {
```

Comment:
nit: Why do we need a new type? Could we just add a `kind` prop to `LinkProps` next to `size`?

Reply:
just imitated the Button comp.

Reply:
For Button we do this because we have an intersection with several other union types  - ButtonAccessibleIconLeftProps, ButtonAccessibleIconRightProps. This component is much simpler. WDYT?

Reply:
no favorites for me. (my type game will also take some time; you know a decade of duck typing rots one's brain :)) so the last commit is more preferable you think?